docs: Improve Build with AI page clarity and structure (#2256)
apify/apify-docs: sources/platform/actors/development/quick-start/build_with_ai.md

toc_max_heading_level: 4
---

import { AGENTS_PROMPT } from "@site/src/utils/agents-prompt";
import PromptButton from "@site/src/components/PromptButton";
import copyForAI from "./images/copy-for-ai.png";
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> 

Author: Yeah, we want to keep vibe coding mainly due to SEO. But if you think that it should be removed, let's remove it.

This guide provides best practices for building new Actors or improving existing ones using AI code generation tools by providing the AI agents with the right instructions and context.

> The methods below are complementary - start with the [AI coding assistant instructions](#ai-coding-assistant-instructions) or [Actor templates with AGENTS.md](#use-actor-templates-with-agentsmd) to get going, then add [Agent Skills](#use-agent-skills) and the [Apify MCP server](#use-apify-mcp-server) to give your assistant more context and better results.

Reviewer: We try to avoid directional language

```suggestion
The methods on this page are complementary - start with the [AI coding assistant instructions](#ai-coding-assistant-instructions) or [Actor templates with AGENTS.md](#use-actor-templates-with-agentsmd) to get going, then add [Agent Skills](#use-agent-skills) and the [Apify MCP server](#use-apify-mcp-server) to give your assistant more context and better results.
```

## Quick start

<Tabs>
<TabItem value="prompt" label="Start with a prompt">

1. Create a directory: `mkdir my-new-actor`
1. Open the directory in _Cursor_, _Claude Code_, _VS Code with GitHub Copilot_, etc.
1. Copy the [AI coding assistant prompt](#ai-coding-assistant-instructions) and paste it into your AI coding assistant
1. Run it, and develop your first Actor with the help of AI

</TabItem>
<TabItem value="template" label="Start with a template">

1. [Install the Apify CLI](/cli/docs/installation) if you haven't already
1. Run `apify create` to initialize an Actor from a [template](https://apify.com/templates) (includes AGENTS.md)
1. Open the project in _Cursor_, _Claude Code_, _VS Code with GitHub Copilot_, etc.
1. Start developing - your AI coding assistant automatically picks up context from AGENTS.md

> 

Reviewer: Same as before

```suggestion
1. [Install the Apify CLI](/cli/docs/installation) if you haven't already.
1. Run `apify create` to initialize an Actor from a [template](https://apify.com/templates) (includes AGENTS.md).
1. Open the project in _Cursor_, _Claude Code_, _VS Code with GitHub Copilot_, etc.
1. Start developing - your AI coding assistant automatically picks up context from AGENTS.md.

```

The command above will guide you through Apify Actor initialization, where you select an Actor Template that works for you. The result is an initialized Actor (with AGENTS.md) ready for development.

## Use Agent Skills

> [Agent Skills](https://github.com/apify/agent-skills) are official Apify skills for Actor development, web scraping, data extraction, automation, etc. They work with Claude Code, Cursor, Codex, Gemini CLI, and other AI coding assistants. Agents are increasingly capable, but often lack the context they need to do real work reliably.

Reviewer: The removed sentence felt like marketing copy without any real value.

```suggestion
[Agent Skills](https://github.com/apify/agent-skills) are official Apify skills for Actor development, web scraping, data extraction, automation, etc. They work with Claude Code, Cursor, Codex, Gemini CLI, and other AI coding assistants.
```
